_maps/shuttles/pirate_Interdyne.dmm

```diff
@@ -491,6 +491,10 @@
 /obj/machinery/atmospherics/components/unary/vent_pump/on{
 	dir = 4
 	},
+/mob/living/simple_animal/bot/medbot/nukie{
+	name = "Dr. Thrax";
```

Comment:
CRINGE blatant video game reference

Reply:
```suggestion
	name = "Dr. Pax";
```

Reply:
how do you know you dont play games

Reply:
because someone brought it up so i googled it

Reply:
@Sealed101 this is your fault

Reply:
it's over